wifiphisher/common/opmode_manager.py

```diff
@@ -0,0 +1,269 @@
+"""
+Define the operation mode manager
+The operation mode will be defined by the arguments and the number of
+avaliable adaptors on the system
+"""
+import sys
+import os
+import logging
+import wifiphisher.common.interfaces as interfaces
+import wifiphisher.common.constants as constants
+import wifiphisher.extensions.handshakeverify as handshakeverify
+
+
+logger = logging.getLogger(__name__)
+
+
+class OpModeManager(object):
```

Comment:
Instead of OpModeManager, I would call this call simply OpMode.

A Manager is something that manages many objects (e.g. opmodes). In this case, there's only one opmode that we use and we stick to it during the runtime.